server/db/index.js

```diff
@@ -1,13 +1,204 @@
 exports.users = require('./users');
 
 exports.getOpenSimulations = function (instructor_token) {
+  /**
+   * http://localhost:3000/api/v1/dashboard
+  */
   return [];
 }
 
 exports.getClosedSimulations = function (instructor_token) {
+  /**
+   * http://localhost:3000/api/v1/dashboard
+  */
   return [];
 }
 
 exports.getDraftedSimulations = function (instructor_token) {
+  /**
+   * http://localhost:3000/api/v1/dashboard
+  */
   return [];
 }
+
+exports.getSimulationSummary = function (instructor_token) {
+  /**
+   * /simulation/:simulation_id/introduction
+   * /simulation/:simulation_id/initial-reflection
+   * /simulation/:simulation_id/initial-action
+  */
+  return "";
+}
+
+exports.getIntialReflectionQuestions = function (token) {
+  // returns an array of questions
+  /**
+   * /simulation/:simulation_id/initial-reflection
+  */
+  return [];
+}
+
+// have to work on this 
+exports.getStudentName = function (token) {
+  /**
+   * /simulation/:simulation_id/initial-reflection/answers
+   * /simulation/:simulation_id/initial-action/answers
+   * /simulation/:simulation_id/stakeholders/answers
+  */
+  return  "";
+}
+
+// have to work on this
+exports.getStudentUID = function (token) {
+  /**
+   * /simulation/:simulation_id/initial-reflection/answers
+   * /simulation/:simulation_id/initial-action/answers
+   * /simulation/:simulation_id/stakeholders/answers
+  */
+  return 0;
+}
+
+exports.getStudentResponses = function (token) {
+  // an array of json with keys 'question', 'answer'.
+  /**
+   * /simulation/:simulation_id/initial-reflection/answers
+   * /simulation/:simulation_id/initial-action/answers
+  */
+  return [];
+}
+
+exports.getQuestion = function (token) {
+  return "";
+}
+
+exports.getAnswer = function (token) {
+  return "";
+}
+
+exports.getIntialActionQuestions = function (instructor_token) {
+   // an array of json with keys 'question', 'choices'.
+   /**
+    * /simulation/:simulation_id/initial-action
+  */
+  return []
+}
+
+exports.getChoices = function (token) {
+  // return : array of string.
+  /**
+   * /simulation/:simulation_id/initial-action
+  */
+  return [];
+}
+
+exports.getSimulationExplanation = function (token) {
+  /**
+    /simulation/:simulation_id/additional-reflection
+    /simulation/:simulation_id/final-decision
+  */
+  return "";
+}
+
+exports.getStudentExplanation = function (token) {
+  /**
+   /simulation/:simulation_id/additional-reflection
+   /simulation/:simulation_id/final-decision
+  */
+  return "";
+}
+
+exports.getStakeHolderSummary = function (token) {
```

Comment:
StakeHolder --> Stakeholder
```suggestion
exports.getStakeholderSummary = function (token) {
```

Reply:
I'll change this write away. Thanks!